packages/govtnz-ds/src/index.ts

```diff
@@ -64,16 +66,49 @@ async function main(
 
   // const silverStripe4Files = await getSilverStripe4Files();
 
-  const releaseSpecItems: ReleaseSpecItem[] = JSON.parse(specString);
+  let releaseSpecItems: ReleaseSpecItem[] = JSON.parse(specString);
   const release: ReleaseItem[] = [];
   let cssVariables: CSSVariablePattern[] = [];
 
+  if (sources !== null) {
+    releaseSpecItems = sources.reduce((result, source) => {
+      const identifier = source.split('@');
+      const sourceId = identifier[0];
+      const version = identifier[1];
+
+      const releaseSpecItem = releaseSpecItems.find(item => {
+        if (item.sourceId !== sourceId) {
+          return false;
+        }
+
+        if (version !== undefined && item.version !== version) {
+          return false;
+        }
+
+        return true;
+      });
+
+      if (releaseSpecItem !== undefined) {
+        result.push(releaseSpecItem);
+      }
+
+      return result;
+    }, []);
+  }
+
```

Comment:
With these changes does a `build:all` still work?

Reply:
Yeah it does - it defaults to using all sources when none are provided via the cli.